### DISCUSSION TITLE
Video MP4 file upload is failing

### DISCUSSION THREAD

**Discussion by Infinil** [pocketbase/pocketbase#5414] [Q&A] [answered]
Trying to upload a video (mp4) through admin console and only one field has to be populated. It fails everytime with `Something went wrong while processing your request.` and logs also don't show the error ones.

**Comment by ganigeorgiev** [comment, marked as answer]
I'm sorry but your post doesn't really make sense to me (_what the `schema cannot be blank` error has to do with the file uploads? is this from an attempt to create a base collection without fields?_).

In any case I've just tested uploading an example mp4 video file (~3mb) locally with PocketBase v0.22.20 and it works correctly for me and I doubt that your issue is PocketBase related (_we don't have special handling for video files_).

Make sure that:
- your file satisfy the collection `file` field max size requirement
- if you have custom hooks ensure that they don't conflict with the file upload
- if you are behind a reverse proxy (nginx, apache, caddy, etc.) ensure that the default request limits (timeout, body size, etc.) are properly configured
- try to start the executable with `--dev` and check the stderr output for any additional details related to the failed request.


I take issues and bug reports seriously but there are just too many missing details about your environment or setup to help you pinpoint the culprit or even understand what actually is the problem.

**If the above doesn't help, please don't treat the issues tracker and discussions as llm chat and try to elaborate a little more on your use case and ideally provide _minimal_ code sample (in case you have custom hooks) and some reproducible steps.**

**Reply by Infinil** [reply]
> I'm sorry but your post doesn't really make sense to me (_what the `schema cannot be blank` error has to do with the file uploads? is this from an attempt to create a base collection without fields?_).
> 
> In any case I've just tested uploading an example mp4 video file (~3mb) locally with PocketBase v0.22.20 and it works correctly for me and I doubt that your issue is PocketBase related (_we don't have special handling for video files_).
> 
> Make sure that:
> 
> * your file satisfy the collection `file` field max size requirement
> * if you have custom hooks ensure that they don't conflict with the file upload
> * if you are behind a reverse proxy (nginx, apache, caddy, etc.) ensure that the default request limits (timeout, body size, etc.) are properly configured
> * try to start the executable with `--dev` and check the stderr output for any additional details related to the failed request.
> 
> I take issues and bug reports seriously but there are just too many missing details about your environment or setup to help you pinpoint the culprit or even understand what actually is the problem.
> 
> **If the above doesn't help, please don't treat the issues tracker and discussions as llm chat and try to elaborate a little more on your use case and ideally provide _minimal_ code sample (in case you have custom hooks) and some reproducible steps.**

Hi, I realized the issue was with `client_max_body_size` in nginx config. Actually the log error I pasted in original issue was the incorrect one and I got it confused because pocketbase wasn't logging anything (due to nginx config). 

Thank you for the help!
